android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -496,6 +501,35 @@ public void onPlayWhenReadyChanged(boolean playWhenReady, int reason) {
         };
         player.addListener(eventListener);
     }
+    private void openSettings() {
+        AlertDialog.Builder builder = new AlertDialog.Builder(themedReactContext);
+        builder.setTitle("Settings");
+        String[] settingsOptions = {"Playback Speed"};
+        builder.setItems(settingsOptions, (dialog, which) -> {
+            if (which == 0) {
+                showPlaybackSpeedOptions();
+            }
+        });
+        builder.show();
+    }
+
+    private void showPlaybackSpeedOptions() {
+        String[] speedOptions = {"0.5x", "1.0x", "1.5x", "2.0x"};
+        AlertDialog.Builder builder = new AlertDialog.Builder(themedReactContext);
+        builder.setTitle("Select Playback Speed");
+
+        builder.setSingleChoiceItems(speedOptions, selectedSpeedIndex, (dialog, which) -> {
+            selectedSpeedIndex = which;
+            float speed = switch (which) {
+                case 0 -> 0.5f;
+                case 2 -> 1.5f;
+                case 3 -> 2.0f;
+                default -> 1.0f;
+            };
+            player.setPlaybackParameters(new PlaybackParameters(speed));
```

Comment:
Can you use setRateModifier please ?
Then we will keep coherent rate management between UI and API behavior